plugins/child/child.js

```diff
@@ -76,13 +76,16 @@ s.cameraVals=function(e){
 }
```

Comment:
child.js is actually not usable at this time, thanks for adding in the subtitle stuff though.